xbmc/video/VideoThumbLoader.cpp

```diff
@@ -436,7 +436,7 @@ bool CVideoThumbLoader::FillLibraryArt(CFileItem &item)
     m_videoDatabase->Open();
     if (m_videoDatabase->GetArtForItem(tag.m_iDbId, tag.m_type, artwork))
       SetArt(item, artwork);
-    else if (tag.m_type == MediaTypeArtist)
+    else if (tag.m_type == "actor" && !tag.m_artist.empty())
```

Comment:
why not specify a define `MediaTypeActor` like we have for all other?

https://github.com/xbmc/xbmc/blob/5737b0c299400a285f33dfb4db964aea688ab1f2/xbmc/media/MediaType.h#L39